CHANGELOG.rst

```diff
@@ -10,8 +10,17 @@ This document describes changes between each past release.
 
 - Remove pypy supports. (#1049)
 
+**Internal changes**
+
+- Permission schema children fields are now set during initialization instead of on
+  deserialization (#1046).
+- Request schemas (including validation and deserialization) are now isolated by method
+  and endpoint type (#1047).
+- Move generic API schemas (e.g TimeStamps and HeaderFields) from `kinto.core.resource.schema`
+  to a sepate file on `kinto.core.schema`.
```

Comment:
> Do you know why we have them there?

Originally ([archeology](https://github.com/mozilla-services/readinglist/blob/master/readinglist/views/article.py)) they were in core because we thought they were common types of fields. But now they're just used in third parties or [plugins](https://github.com/mozilla-services/kinto-fxa/blob/master/kinto_fxa/views/relier.py#L19). We can deprecated them, no worries.